controllers/ovndbcluster_controller.go

```diff
@@ -533,9 +533,16 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	//
 	// Ensure the ovndbcluster headless service Exists
 	//
+	headlessServiceLabels := map[string]string{
```

Comment:
can just extend serviceLabels here instead of duplicating.

Reply:
Extending map from serviceLabels would modify servicelabels, we could use maps.Clone https://pkg.go.dev/maps

Reply:
i was considering like using something ilke https://github.com/openstack-k8s-operators/lib-common/blob/c9467a8fbbfc3e244207cd3d435975aca298d33d/modules/common/util/map.go#L86